object-manipulation-2/src/defaults.js

```diff
@@ -0,0 +1,8 @@
+/* exported defaults */
+function defaults(target, source) {
+  for (var keys in source) {
+    if (Object.prototype.hasOwnProperty.call(source, keys) && !Object.prototype.hasOwnProperty.call(target, keys)) {
```

Comment:
This is way over what we have to do but good job. The simpler solution would be `if (typeof target[key] === 'undefined')`